newdle/api.py

```diff
@@ -270,6 +270,10 @@ def create_newdle(title, duration, timezone, timeslots, participants, private, n
         lambda p: {
             'creator': newdle.creator_name,
             'title': newdle.title,
+            'participant_name': p.name,
+            'newdle_link': url_for(
+                'newdle', code=newdle.code + '/summary', _external=True
```

Comment:
Please do not abuse the `code` arg like this. we alraedy have a `newdle_summary` endpoint that generates a link for the summary.

Reply:
Not sure how to do it, can you help me?

Reply:
`url_for('newdle_summary', ...)`